support-frontend/assets/helpers/abTests/abtestDefinitions.ts

```diff
@@ -147,4 +147,25 @@ export const tests: Tests = {
 		targetPage: pageUrlRegexes.contributions.allLandingPagesAndThankyouPages,
 		excludeCountriesSubjectToContributionsOnlyAmounts: false,
 	},
+	newOneTimeCheckout: {
+		variants: [
+			{
+				id: 'control',
+			},
+			{
+				id: 'variant',
+			},
+		],
+		audiences: {
+			ALL: {
+				offset: 0,
+				size: 1,
+			},
+		},
+		isActive: false,
+		referrerControlled: false, // ab-test name not needed to be in paramURL
+		seed: 4,
+		targetPage: pageUrlRegexes.contributions.allLandingPagesAndThankyouPages,
+		excludeCountriesSubjectToContributionsOnlyAmounts: false,
```

Comment:
@andrewHEguardian I think `excludeCountriesSubjectToContributionsOnlyAmounts` should be `true` for this test, users who are subject to contributions only amounts see a different checkout flow (the old 2-step checkout), it's easier to exclude them than include them. Including them means we should update their checkout flow to also use the new one-time-checkout if they're in the variant.